src/Http/Http.Abstractions/src/HttpRequest.cs

```diff
@@ -115,11 +115,24 @@ public abstract class HttpRequest
     /// <summary>
     /// Gets or sets the request body as a form.
     /// </summary>
+    /// <exception cref="System.InvalidOperationException">
+    ///     incorrect content-type.
+    /// </exception>
+    /// <remarks>
+    ///     <para>
+    ///     Invoking this property could result in thread exhaustion since it's wrapping an asynchronous implementation.
+    ///     To prevent this the method <see cref="ReadFormAsync" /> can be used.
```

Comment:
```suggestion
    ///     To prevent this the method <see cref="ReadFormAsync(CancellationToken)"/> can be used.
```

I think this should do it?

Reply:
Why do you need the parameter list if there's only one overload?  It seems to work without in local tests.

Reply:
Oh, I see, it was supposed to be the one in src/Http/Http.Features/src/IFormCollection.cs, not in this file.  Yes, either add `(CancellationToken)` or remove `()`.